web/renderer/components/DataTable/Table/HeadCell.tsx

```diff
@@ -34,25 +34,35 @@ export default function HeadCell({
 }: Props) {
   const [showDropdown, setShowDropdown] = useState(false);
   const showCollapseCellButton = isLongContentType(col.type, col.name);
+  const dataCy = `${isMobile ? "mobile" : "desktop"}-db-data-table-column-${col.name}`;
+  const dataCyDropDown = `${dataCy}-dropdown`;
+  const dataCyDropDownSortButton = `${dataCyDropDown}-sort`;
   return (
     <th
       className={cx(css.cell, {
         [css.active]: showDropdown,
       })}
-      data-cy={`${isMobile ? "mobile-" : "desktop-"}db-data-table-column-${
-        col.name
-      }`}
+      data-cy={dataCy}
     >
       {col.name}
       {col.isPrimaryKey && <FiKey className={css.key} />}
       <CellDropdown
         showDropdown={showDropdown}
         setShowDropdown={setShowDropdown}
         buttonClassName={css.menu}
+        data-cy={dataCyDropDown}
       >
-        <SortButton dir="ASC" col={col} />
-        <SortButton dir="DESC" col={col} />
-        <SortButton col={col} />
+        <SortButton
+          dir="ASC"
+          col={col}
+          dataCy={`${dataCyDropDownSortButton}`}
+        />
+        <SortButton
+          dir="DESC"
+          col={col}
+          dataCy={`${dataCyDropDownSortButton}`}
+        />
+        <SortButton col={col} dataCy={`${dataCyDropDownSortButton}`} />
```

Comment:
Don't need the template strings for these
```suggestion
          dataCy={dataCyDropDownSortButton}
        />
        <SortButton
          dir="DESC"
          col={col}
          dataCy={dataCyDropDownSortButton}
        />
        <SortButton col={col} dataCy={dataCyDropDownSortButton} />
```